troubleshooting-2.Rmd

```diff
@@ -135,17 +135,23 @@ Without using `group_by()`, I want to find out how many movie reviews there have
 
 ```{r}
 ### ERROR HERE ###
-movieLens %>%
-  tally(year)
+#movieLens %>%
+#  tally(year)
+#Changed to 
+movieLens %>%   # Tally is used for grouped data, count is a short-hand for group_by()
+  count(year)
 ```
 
 Both `count()` and `tally()` can be grouped by multiple columns. Below, I want to count the number of movie reviews by title and rating, and sort the results.
 
 ```{r}
 ### ERROR HERE ###
+#movieLens %>%
+#  count(c(title, rating), sort = TRUE)
+# changed to: 
 movieLens %>%
-  count(c(title, rating), sort = TRUE)
-```
```

Comment:
The fixes look good, but without these three backticks on line 148 the codeblock doesn't end, and I get an error as knitr tries to interpret line 156 as code.
The same error occurs at line 169.